tests/envs/dummy_env.py

```diff
@@ -0,0 +1,17 @@
+import gym
+
+
+class DummyEnv(gym.Env):
+    def __init__(self):
+        super().__init__()
+        self.act_space = gym.spaces.Discrete(2)
+        self.obs_space = gym.spaces.Discrete(2)
+
+    def reset(self):
```

Comment:
For consistency, can you update the signature here to follow the new conventions?

Reply:
I assume you mean returning info with reset?

Reply:
Yep, and the arguments